src/Apps/Order/redirects.tsx

```diff
@@ -186,3 +205,39 @@ export const redirects: RedirectRecord<{
     },
   ],
 }
+
+graphql`
```

Comment:
Interesting! This doesn't seem to be a GraphQL query that's part of any Relay container/config (usually where it's used).

It's just sort of...hanging out in this file. Is that a Relay compiler thing which will generate everything for strings like this it encounters, and you don't need a `const redirectFragment = graphql`...`?

Reply:
Yep, exactly!

Reply:
The Relay babel plugin will replace any `graphql` tags with a function that imports the artefact that was emitted by `relay-compiler`. So in this case it gets translated to an unused function like:

```js
function order() {
  return require("__generated__/redirects_order.graphql");
}
```

i.e. It only adds a tiny amount of code to the overall runtime and the function never gets invoked. Perhaps a good tree-shaking bundler setup could even strip these out, if it’s able to figure out that it’s dead code, but that’s not something I’m immediately concerned with.